test/lib/run-build-lambda.js

```diff
@@ -4,75 +4,56 @@ const fs = require('fs-extra');
 const json5 = require('json5');
 const { glob } = require('@vercel/build-utils');
 
-function runAnalyze(wrapper, context) {
-  if (wrapper.analyze) {
-    return wrapper.analyze(context);
-  }
-
-  return 'this-is-a-fake-analyze-result-from-default-analyze';
-}
-
-async function runBuildLambda(inputPath) {
-  const inputFiles = await glob('**', inputPath);
-  const nowJsonRef = inputFiles['vercel.json'] || inputFiles['now.json'];
-
-  if (typeof expect !== 'undefined') {
-    expect(nowJsonRef).toBeDefined();
-  }
-  const nowJson = json5.parse(await fs.readFile(nowJsonRef.fsPath, 'utf8'));
-  const build = nowJson.builds[0];
-
-  if (typeof expect !== 'undefined') {
-    expect(build.src.includes('*')).toBeFalsy();
-  }
-  const entrypoint = build.src.replace(/^\//, ''); // strip leftmost slash
-
-  if (typeof expect !== 'undefined') {
-    expect(inputFiles[entrypoint]).toBeDefined();
-  }
-  inputFiles[entrypoint].digest =
-    'this-is-a-fake-digest-for-non-default-analyze';
-  const wrapper = require(build.use);
-
-  const analyzeResult = runAnalyze(wrapper, {
-    files: inputFiles,
-    entrypoint,
-    config: build.config,
-  });
```

Comment:
**question (non-blocking):** Was `analyzeResult` not being used anywhere?

Reply:
Yes, Builders used to have an `analyze()` function but that's no longer the case.